prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/health-auth-org-info-page/health-auth-org-info-page.component.html

```diff
@@ -4,131 +4,207 @@
   <app-page-header>{{ healthAuthority?.name }} Organization Information</app-page-header>
 
   <div class="row">
-    <div class="col-6">
+    <div class="col-md-6 col-sm-12">
 
-      <div class="pb-3">
-        <app-page-subheader>
+      <app-page-section>
+        <app-page-subheader [divider]="false">
           <ng-container appPageSubheaderTitle>
-            <mat-icon class="mr-2">business</mat-icon>
+            <mat-icon class="mr-2">location_city</mat-icon>
```

Comment:
Is there a reason this was changed from "business"?

Reply:
I based this "overview page" off of the site overview admin page and for the organization information we use that icon so just stuck with it.

Reply:
But we use the `business` icon here?

![image](https://user-images.githubusercontent.com/7586665/152853597-bde5934e-b87e-4492-ba42-fa81c698b255.png)

Reply:
business it is